composables/solr.js

```diff
@@ -0,0 +1,67 @@
+
+
+export async function queryIndex(searchQuery) {
+    const { cbdApi }       = useRuntimeConfig().public;
+    const result = await useFetch(`https://api.cbd.int/api/v2013/index/select`, { method: 'POST', body: searchQuery })
```

Comment:
not missing  `${cbdApi}`?
```suggestion
    const result = await useFetch(`${cbdApi}/api/v2013/index/select`, { method: 'POST', body: searchQuery })
```